### DISCUSSION TITLE
Installing Solito

### DISCUSSION THREAD

**Discussion by nadleeh00** [nandorojo/solito#343] [Q&A] [answered]
Is there anyone who has actually installed Solito from 0? From a plain Next.js project?
I've been trying, but it doesn't seem possible. It also seems like everyone is just copying the starters.

I looks as though it's kind of glossed over.

**Comment by JWesorick** [comment, marked as answer]
You should start from a starter and pull you existing project in. With it being a mono-repo structure and all of the special configuration going on it is much easier.
